apps/web/src/components/new-safe/create/steps/ReviewStep/index.tsx

```diff
@@ -299,13 +305,29 @@ const ReviewStep = ({ data, onSubmit, onBack, setStep }: StepRenderProps<NewSafe
 
     gtmSetChainId(chain.chainId)
 
+    trackEvent(CREATE_SAFE_EVENTS.CREATED_SAFE, {
+      [MixPanelEventParams.SAFE_ADDRESS]: safeAddress.toLowerCase(),
```

Comment:
I see that you do this toLowerCase call everywhere we call the function. I guarantee 100% that someone is going to forget that toLowerCase is necessary. Can't we move this toLowerCase somewhre inside of trackEvent so that it is automatic. As someone who uses the trackEvent function I shouldn't need to know of the internal details of the function.

Reply:
I looked in the code. I guess I would do this in safeMixPanelTrack. If the props have a SAFE_ADDRESS cast it to lowercase just in case.